[2025-10-29T15:50:07.551+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-29T15:50:07.575+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flood_prediction_pipeline.send_slack_alert scheduled__2025-10-29T15:45:00+00:00 [queued]>
[2025-10-29T15:50:07.585+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flood_prediction_pipeline.send_slack_alert scheduled__2025-10-29T15:45:00+00:00 [queued]>
[2025-10-29T15:50:07.587+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-29T15:50:07.601+0000] {taskinstance.py:2330} INFO - Executing <Task(SlackWebhookOperator): send_slack_alert> on 2025-10-29 15:45:00+00:00
[2025-10-29T15:50:07.614+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1770) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-29T15:50:07.614+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'flood_prediction_pipeline', 'send_slack_alert', 'scheduled__2025-10-29T15:45:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/flood_prediction_dag.py', '--cfg-path', '/tmp/tmpwacnxbjm']
[2025-10-29T15:50:07.616+0000] {standard_task_runner.py:63} INFO - Started process 1780 to run task
[2025-10-29T15:50:07.617+0000] {standard_task_runner.py:91} INFO - Job 36: Subtask send_slack_alert
[2025-10-29T15:50:07.682+0000] {task_command.py:426} INFO - Running <TaskInstance: flood_prediction_pipeline.send_slack_alert scheduled__2025-10-29T15:45:00+00:00 [running]> on host db3c1202c602
[2025-10-29T15:50:07.785+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flood_prediction_pipeline' AIRFLOW_CTX_TASK_ID='send_slack_alert' AIRFLOW_CTX_EXECUTION_DATE='2025-10-29T15:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-29T15:45:00+00:00'
[2025-10-29T15:50:07.788+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-29T15:50:07.817+0000] {base.py:84} INFO - Using connection ID 'slack_webhook_default' for task execution.
[2025-10-29T15:50:08.356+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-29T15:50:08.363+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=flood_prediction_pipeline, task_id=send_slack_alert, run_id=scheduled__2025-10-29T15:45:00+00:00, execution_date=20251029T154500, start_date=20251029T155007, end_date=20251029T155008
[2025-10-29T15:50:08.395+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-29T15:50:08.408+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-29T15:50:08.412+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
